public/components/content-list-drawer/content-list-drawer.html

```diff
@@ -1,10 +1,10 @@
 <tr class="content-list-drawer content-list-drawer--hidden" ng-class="{'content-list-drawer--trashed': contentItem.item.trashed}">
     <td colspan="100" class="content-list-drawer__full-width">
 
-        <div class="drawer">
-            <div class="drawer__section drawer__section--wide">
-                <div class="drawer__header">
-                    <i class="drawer__icon" wf-icon="furniture"/>
+        <div class="drawer" wf-content-list-drawer-accordion wf-open-section="furniture">
```

Comment:
It was my original idea for an arribute based directive for the toggling. I'm gonna pull it out now as I moved this up to the drawer controller in #77